src/tests/test_adapters.py

```diff
@@ -156,3 +157,137 @@ def test_text_integration_adapter(client: vecs.Client) -> None:
     with pytest.raises(ArgError):
         # if the cardinality changes due to adapter pre-processing, raise an error
         docs.query(data="I split \n\n into multiple records \n\n not good")
+
+
+def test_markdown_chunker_normal_headings() -> None:
+    chunker = MarkdownChunker(skip_during_query=True)
+    res = [
+        x
+        for x in chunker(
+            [
+                (
+                    "1",
+                    "# heading 1\n## heading 2\n### heading 3\n#### heading 4\n##### heading 5\n###### heading 6",
```

Comment:
adding at least 1 section with some words in it here would be a good test for the md parser